mosaic-runtime/src/commonTest/kotlin/com/jakewharton/mosaic/MosaicTest.kt

```diff
@@ -24,4 +29,51 @@ class MosaicTest {
 			""".trimMargin().replaceLineEndingsWithCRLF(),
 		)
 	}
+
+	@Test fun counter() {
+		runTest {
```

Comment:
```suggestion
	@Test fun counter() = runTest {
```
`runTest` returns a `Promise<Unit>` on JS that the testing harness will `await()`. Otherwise the test won't actually run (or, at least, won't run past the first suspension point).

Reply:
Probably the first time I wrote `runTest` without `return` and overlooked this behavior, thank you!